plugins/tekton/src/components/pipeline-topology/PipelineVisualization.test.tsx

```diff
@@ -16,8 +17,13 @@ jest.mock('@backstage/plugin-catalog-react', () => ({
   }),
 }));
 
+jest.mock('@backstage/core-plugin-api', () => ({
+  ...jest.requireActual('@backstage/core-plugin-api'),
+  useRouteRef: () => () => '/xyz',
+}));
+
 describe('PipelineVisualization', () => {
-  it('should render the pipeline run visualization when pipelineRun exists', async () => {
+  it('should render the latest pipeline run visualization when pipelineRun exists', async () => {
```

Comment:
component should be declarative, and should just render PLR without the notion of this being the latest or old one.